sdk/spring/spring-cloud-azure-service/src/main/java/com/azure/spring/cloud/service/implementation/identity/api/AuthProperty.java

```diff
@@ -8,36 +8,46 @@
 
 public enum AuthProperty {
 
-    CLIENT_ID("azure.clientId"),
-    CLIENT_SECRET("azure.clientSecret"),
-    CLIENT_CERTIFICATE_PATH("azure.clientCertificatePath"),
-    CLIENT_CERTIFICATE_PASSWORD("azure.clientCertificatePassword"),
-    USERNAME("azure.username"),
-    PASSWORD("azure.password"),
-    MANAGED_IDENTITY_ENABLED("azure.managedIdentityEnabled"),
-    AUTHORITY_HOST("azure.authorityHost", AzureAuthorityHosts.AZURE_PUBLIC_CLOUD),
-
-    TENANT_ID("azure.tenantId"),
-    CLAIMS("azure.claims"),
-    SCOPES("azure.scopes"),
-
-    // TODO define this here?,
-    GET_TOKEN_TIMEOUT("azure.accessTokenTimeoutInSeconds"),
-    TOKEN_CREDENTIAL_PROVIDER_CLASS_NAME("azure.tokenCredentialProviderClassName"),
-    TOKEN_CREDENTIAL_BEAN_NAME("azure.tokenCredentialBeanName"),
-    CACHE_ENABLED("azure.cacheEnabled")
+    CLIENT_ID("azure.clientId", "Client ID to use when performing service principal authentication with Azure.", false),
+    CLIENT_SECRET("azure.clientSecret", "lient secret to use when performing service principal authentication with Azure.", false),
+    CLIENT_CERTIFICATE_PATH("azure.clientCertificatePath", "Path of a PEM certificate file to use when performing service principal authentication with Azure.", false),
+    CLIENT_CERTIFICATE_PASSWORD("azure.clientCertificatePassword","Password of the certificate file.",false),
+    USERNAME("azure.username", "Username to use when performing username/password authentication with Azure.", false),
+    PASSWORD("azure.password", "Password to use when performing username/password authentication with Azure.", false),
+    MANAGED_IDENTITY_ENABLED("azure.managedIdentityEnabled", "Whether to enable managed identity to authenticate with Azure.", false),
+    AUTHORITY_HOST("azure.authorityHost", AzureAuthorityHosts.AZURE_PUBLIC_CLOUD, "AzureAuthority Host", true),
+
+    TENANT_ID("azure.tenantId", "Tenant ID for Azure resources.", true),
+    CLAIMS("azure.claims", "Claims for Azure resources.", false),
+    SCOPES("azure.scopes", "Scopes for Azure resources.", false),
+
+    GET_TOKEN_TIMEOUT("azure.accessTokenTimeoutInSeconds", "Max time to get AccessToken", false),
+    TOKEN_CREDENTIAL_PROVIDER_CLASS_NAME("azure.tokenCredentialProviderClassName", "TokenCredentialProvider class name contains full path.", false),
```

Comment:
`TokenCredentialProvider class name contains full path` what does the contains full path mean?

Reply:
I will update it.